exercises/concept/savings-account/.meta/exemplar.rb

```diff
@@ -22,6 +22,7 @@ def self.annual_balance_update(balance)
   end
 
   def self.years_before_desired_balance(current_balance, desired_balance)
+    raise ArgumentError, 'Current balance cannot be negative' if current_balance.negative?
```

Comment:
```suggestion
    raise ArgumentError, 'Current balance must be positive' if current_balance <= 0
```

Because 0 will give the same problem.

Reply:
Maybe this reads even nicer?

```suggestion
    raise ArgumentError, 'Current balance must be positive' unless current_balance.positive?
```

Reply:
@apmiller108 I do like this version better.

But we should also weigh, like @iHiD asked about, if this is a product of the solution, rather than a problem with the exercise itself.  In other words, it may be a mentoring point instead of a test (potentially out of scope of the exercise).

Should it be a mentor note instead?

Reply:
Thanks for the feedback and good suggestions. I've gone ahead and made the changes. As to whether or not this should be a mentor note, as @iHiD pointed out, I actually didn't know there were mentor notes (first PR to exercism here). Is that the `.docs/hints.md`? I suppose this case might be more appropriate as a note; however, it does make for a good learning topic and I see many community solutions don't consider this case -- those that didn't receive mentoring, perhaps. But, no, this isn't a problem with the exercise itself, rather an edge case where the caller can get into trouble with balances less than zero, which are expected to exist.